docs/docs/rest-api/public/api/v2/types/pod-instance-status.raml

```diff
@@ -0,0 +1,62 @@
+#%RAML 1.0 DataType
+type: object
+additionalProperties: false
+properties:
+  id:
+    type: !include name.raml
+    description: |
+      Unique ID of this pod instance in the cluster.
+      TODO(jdef) Probably represents the Mesos executor ID.
+  status:
+    type: string
+    enum: [ PENDING, STAGING, STABLE, DEGRADED, TERMINAL ]
```

Comment:
Like @meichstedt said, I would introduce a PODStatus with this 14 status and pod status change on defined task status changes. Some POD status changes are triggered if all tasks are in a defined status and some POD status changes are triggered if one task has a certain status.